app/controllers/auth0_controller.rb

```diff
@@ -5,7 +5,16 @@ def callback
     session[:userinfo] = request.env["omniauth.auth"]
 
     # Redirect to the URL you want after successful auth
-    redirect_to "/dashboard"
+
+    user = User.find_by(email: session[:userinfo][:info][:email])
```

Comment:
Could we use `current_user` and possibly save the call to the database?

Reply:
Apparently we can!